posts/2024-01-31-OutreachyInternshipJourney/index.qmd

```diff
@@ -0,0 +1,150 @@
+
+## Bioconductor
+The Bioconductor project aims to develop and share open source software for precise and repeatable analysis of biological data. The Bioconductor community fosters an inclusive and collaborative community of developers and data scientists. 
+
+Bioconductor uses the R statistical programming language and is open source. The Bioconductor community values an open approach to science and promotes:
+
+1. Sharing of ideas, code, software, and expertise.
+
+2. Collaboration.
+
+3. Diversity and Inclusivity.
+
+4. A kind and welcoming environment.
+
+5. Community contributions.
+
+The success of the Bioconductor project has also grown a welcoming and inclusive community made up of a core team, package developers, and users of all backgrounds and experience levels. 
+
+Bioconductor release version is updated twice each year and is appropriate for most users. There is also a development version, to which new features and packages are added before incorporation into the release. 
+
+Bioconductor is an outstanding project and the brand goals are;
+1. To provide widespread access to a broad range of powerful statistical and graphical methods for the analysis of genomic data.
+2. To provide a common software platform that enables the rapid development and deployment of extensible, scalable, and interoperable software.
+3. To further scientific understanding by producing high-quality documentation and reproducible research.
+4. To train researchers on computational and statistical methods for the analysis of genomic data. 
+
+## Microbiome Study Curation
+The microbiome study curation project is a bioconductor project that uses BugSigDB, a database of manually curated microbial signatures obtained from the results of published microbiome differential abundance studies. Differential abundance studies are studies that compare the relative abundance of different microbial species in different samples in hosts of interest (Humans, animals, and environmental conditions). In humans, they help to understand how different geographical and environmental factors can lead to changes in the microbiome, thereby increasing the likelihood of developing conditions such as diabetes, obesity, cardiovascular and neurological diseases, allergies, and inflammatory bowel disease. This information helps to give insights into the role of specific microbes in different biological processes or diseases.
+The Microbiome Study Curation project entails curating new studies on bugsigDB, reviewing studies that have already been curated, and analyzing BugSigDB data using R. To learn more about the project and how to contribute to it, check out my blog post [here](https://kunmiwrites.wordpress.com/2024/01/22/bugsigdb-101-a-beginners-guide-to-exploring-microbial-signatures/).
```

Comment:
Rather than saying `my blog post` since there are three authors, it would be better to say `Esther's blog post`.

Also rather than saying `here`, you could just make "Esther's blog post" the link.